config/application.yml

```diff
@@ -1,7 +1,13 @@
 default: &default
+  TZ: UTC
   DEEP_THOUGHT__LOG_LEVEL: INFO
   DEEP_THOUGHT__RAILS_LOG_TO_STDOUT: true
   DEEP_THOUGHT__DB_URI: mongodb://localhost:27017/deep_thought
+  DEEP_THOUGHT__PROTOCOL: http
+  DEEP_THOUGHT__PORT: 3000
+  DEEP_THOUGHT__HOSTNAME: deepthought.localhost.com
+  DEEP_THOUGHT__AUTH__JWT_SECRET_KEY: <%= SecureRandom.urlsafe_base64(128) %>
+  DEEP_THOUGHT__AUTH__EMAIL_DOMAIN_PATTERN: .*
```

Comment:
Como esses valores se comportam em produção ?
Existe um fallback para pegar da variável de ambiente mesmo do `SO` ou é o que está setado aqui que vale para a aplicação ?

Reply:
Esse aquivo config/application.yml define as variáveis padrões, se alguma dessas variáveis for definida pelo OS, então a variável do OS ganha prioridade.

Você pode ver em a classe em lib/environment.rb e a utilização em config/application.rb:29

Reply:
:+1: